website/templates/public/register.mako

```diff
@@ -92,57 +58,20 @@
-                        </tr>
-                    </table>
-                %elif campaign == "scielo-preprints":
-                    <table style="border-collapse: separate; border-spacing: 30px 0; margin-top: 20px;  margin-bottom: 10px;">
-                        <tr>
-                            <td><img src="/static/img/preprint_providers/scielo-login.png" style="width: 150px; padding: 0 10px 0 10px;" /></td>
-                            <td><h3>Create a free OSF account to contribute to SciELO</h3></td>
-                        </tr>
-                    </table>
-                %elif campaign == "agrixiv-preprints":
-                    <table style="border-collapse: separate; border-spacing: 30px 0; margin-top: 20px; margin-bottom: 10px;">
-                        <tr>
-                            <td><img src="/static/img/preprint_providers/agrixiv-login.svg" style="width: 150px; padding: 0 10px 0 10px;" /></td>
-                            <td><h3>Create a free OSF account to contribute to AgriXiv</h3></td>
-                        </tr>
-                    </table>
-                %elif campaign == "bitss-preprints":
-                    <table style="border-collapse: separate; border-spacing: 30px 0; margin-top: 20px;  margin-bottom: 10px;">
-                        <tr>
-                            <td><img src="/static/img/preprint_providers/bitss-login.png" style="width: 150px; background: #a9a9a9; padding: 0 10px 0 10px;" /></td>
-                            <td><h3>Create a free OSF account to contribute to BITSS</h3></td>
-                        </tr>
-                    </table>
-                %elif campaign == "lawarxiv-preprints":
-                    <table style="border-collapse: separate; border-spacing: 30px 0; margin-top: 20px;  margin-bottom: 10px;">
-                        <tr>
-                            <td><img src="/static/img/preprint_providers/lawarxiv-logo.png" style="width: 150px; padding: 0 10px 0 10px;" /></td>
-                            <td><h3>Create a free OSF account to contribute to LawArXiv</h3></td>
-                        </tr>
-                    </table>
-                %else:
+                %elif campaign not in preprint_campaigns.keys():
                     <h3 class="m-b-lg"> Create a free account </h3>
+                %else:
+                    %for provider in preprint_campaigns.keys():
+                        %if campaign == provider:
+                            <table style="border-collapse: separate; border-spacing: 30px 0; margin-top: 20px;  margin-bottom: 10px;">
+                                <tr>
+                                    <!-- Path to be changed to /wherever preprint assets are/id/square_color_no_transparent.png -->
+                                    <td><img src="${"/static/img/preprint_providers/" + preprint_campaigns[provider]['id'] + "-login.png"}" style="width: 200px; margin-top: 15px" /></td>
```

Comment:
I should probably make these images 200px, then.

Also, call these images `/static/preprint_assets/<provider_name>/square_color_no_transparent.png` and we'll need to move files into there. I'll try to make these while you update (but you'll have your changes done sooner).